site/content/internal/infrastructure/kubernetes/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Kubernetes Troubleshooting"
+date: 2018-11-07T15:24:42+01:00
+weight: 10
+---
+
+This page helps developers access and perform any type of maintenance in the Production Mattermost Kubernetes Cluster, which is running on AWS using EKS.
+
+## Set up a local environment to access Kubernetes (K8s)
```

Comment:
I updated this section slightly, please help confirm changes look okay (https://github.com/mattermost/mattermost-developer-documentation/pull/185/commits/aad5c3fb6a5e3f8520eb3d7be923e6af7e49531e)